NEWS

```diff
@@ -140,31 +138,31 @@ CHANGES WITH 250 in spe:
           [Condition|Assert][Memory|CPU|IO]Pressure= have been added to make a
           unit skip/fail activation if the system's (or a slice's) memory/cpu/io
           pressure is above the configured threshold, using the kernel PSI
-          feature. Fore more details see systemd.unit.5 and
+          feature. For more details see systemd.unit(5) and
           https://www.kernel.org/doc/html/latest/accounting/psi.html
 
         * The combination of ProcSubset=pid and ProtectKernelTunables=yes and/or
           ProtectKernelLogs=yes can now be used.
 
-        * The default maximum number of inodes for /dev has been doubled, from
-          64k to 128k.
+        * The default maximum numbers of inodes have been raised from 64k to 1m
+          for /dev, and from 400k to 1m for /tmp.
```

Comment:
I don't know if it matters, but the SI prefix for 10^6 is M and not m. The latter is for milli, 10^-3.

Reply:
0.001 inodes per FS would be quite the regression!

Reply:
I used "m" because that's what's used in the parameter. But yeah, I can see how it can be misleading. I'll fix that up later.